packages/snaps-cli/src/config.ts

```diff
@@ -109,6 +109,13 @@ export type SnapBrowserifyConfig = {
      */
     port?: number;
 
+    /**
+     * The host to listen the server on.
+     *
+     * @default localhost
+     */
+    host?: number;
```

Comment:
Shouldn't this be a `string`?

```suggestion
    host?: string;
```